Bank_account.js

```diff
@@ -0,0 +1,29 @@
+function Creat_account(name, birthday, money) {
+  this.name = name;
+  this.birthday = birthday;
+  this.money = money;
+
+  this.getInfo = () => `Name: ${this.name}, Age: ${calculateAge(this.birthday)}, Amount: ${this.money}$`;
+  
+  const history = [`Initial: ${money}`];
+
+  this.addMoney = (amount, source) => {
+    this.money = this.money + amount;
```

Comment:
краще записати коротше
```suggestion
    this.money += amount;
```